__tests__/db.test.js

```diff
@@ -0,0 +1,31 @@
+const app = require("../app");
+const request = require('supertest');
+const db = require("../db/connection")
+const seed = require("../db/seeds/seed")
+const testData = require("../db/data/test-data/index")
+
+beforeAll(() => seed(testData));
+afterAll(()=> db.end())
+
+describe("GET /api/topics", () =>{
+    test("Nonsense requests return a 404", () =>{
```

Comment:
To refactor, I'd consider putting this in its own describe block as it would be a test covering endpoints in general rather than specific to this one